userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -292,6 +274,36 @@ of the following:
 [wordmark]: https://en.wikipedia.org/wiki/Wordmark
 [your logo]: /docs/adding-content/iconsimages/#add-your-logo
 
+### Light/Dark mode menu
+
+To enable the display of a Light-/[Dark-mode] menu in the navbar, set
+`params.ui.showLightDarkModeMenu` to true in your project's configuration file.
+The dropdown menu appears at the extreme right, just before of the [search box],
```

Comment:
maybe better: at the *far* right?

Reply:
Actually, I'll simply drop "extreme". WDYT?